dashboard/test/ui/features/textToSpeech.feature

```diff
@@ -40,10 +40,10 @@ Scenario: Listen to TTS Audio in CSF
 
 Scenario: Listen to TTS Audio in CSD
   Given I am a student
-  And I am on "http://studio.code.org/s/csd3/stage/4/puzzle/4?enableExperiments=CSDTTS"
-  And I rotate to landscape
+  And I am on "http://studio.code.org/s/allthettsthings/stage/1/puzzle/1?enableExperiments=CSDTTS"
   And I wait for the page to fully load
-
+  And I close the instructions overlay if it exists
```

Comment:
nit: "wait for the page to fully load" [actually includes this step](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/ui/features/step_definitions/steps.rb#L103-L109)